src/App.tsx

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable no-alert */
+import React from 'react';
+import { connect } from 'react-redux';
+
+import styles from './assets/stylesheets/index.scss';
+import GameBoard from './components/GameBoard/GameBoard';
+// import { } from './redux/actions';
+// import { State } from './types/states.model';
+
+class App extends React.Component {
+  constructor(props: any) {
+    super(props);
+    this.state = {
+
+    };
+  }
+
+  render() {
+    return (
+      <React.Fragment>
+        <h1 className={styles['title']}>2048</h1>
+        <div className={styles['game-container']}>
+          <GameBoard />
+        </div>
+      </React.Fragment>
+    );
+  }
+}
+
+const mapStateToProps = (/* state: State */): any => ({
+
+});
+
+const mapDispatchToProps = {
+
+};
+
+export default connect(mapStateToProps, mapDispatchToProps)(App);
```

Comment:
If you don't use redux in your component, you shouldn't connect it at all. 
It could make some performance issues in future development.